src/components/cart/CartItem.ts

```diff
@@ -27,6 +28,7 @@ class CartItem extends Base<HTMLDivElement> {
 
     const { cart } = store.getOptions();
     this.currency = cart?.currency || DEFAULT_CURRENCY;
+    this.currencyBehind = cart?.currencyBehind ?? false;
```

Comment:
For consistency I would create a constant `DEFAULT_CURRENCY_POSITION` and set it to `'start'`, so this would be:

```
this.currencyPosition = cart?.currencyPosition || DEFAULT_CURRENCY_POSITION;
```